### ISSUE TITLE
How to get the BT pin?

### ISSUE THREAD

**Issue by vaderj** [meshtastic/Meshtastic-device#137]
OS: Win 10 1909
Using esphome-flash-1.2.0-windows-x64.exe, after completing the writing of : 

firmware-ttgo-lora32-v2-US-0.6.3.bin 

to a TTGO T-Beam, I receive the following output (after clicking "View Logs")

![image](https://user-images.githubusercontent.com/5408195/82129457-d6457500-9777-11ea-9ea4-74d27e57a495.png)

Am I looking in the wrong spot? I looked through the forum and I didn't see anything that looked related

**Comment by vaderj** [comment]
I realized that I had not written the correct firmware, but even after writing the tbeam firmware, the BT Pin is NOT printed to the diag output that is visible from the ESPHome util.

I tried looking at the port via Arduino Serial Monitor but none of the speeds resulted in readable text.

I was just trying to verify the hardware booted before I soldered stuff up, and just applying manual pressure with the screen allowed me to see the Bluetooth Pin and verify the devices, at least BT and firmware, worked.

Thats for such an awesome project! I hope to be able to contribute in some form soon!
